mix.exs

```diff
@@ -33,7 +33,7 @@ defmodule Dogma.Mixfile do
   defp deps do
     [
       # Test framework
-      {:shouldi, only: :test},
+      {:shouldi, only: ~w(dev test)a},
```

Comment:
It looks like Coveralls is meant to be available in the dev environment, but it depends on ShouldI. Adding ShouldI to dev means that Coveralls can be run in dev too.